### PR TITLE
Couple of minor improvements

### PR DESCRIPTION
**Pull request by Gonzih**
Hello,
I'm using your library in my pet project and I have couple of improvements in mind.
No breaking chages, I can split that in 2 pull requests if you like.
Also feedback is welcome.

Thanks!

### CODE REVIEW

File: src/feedparser_clj/core.clj [hunk 1]
```diff
@@ -94,13 +94,20 @@
...
 (defn parse-feed "Get and parse a feed from a URL"
   ([feedsource]
-     (parse-internal (new XmlReader (if (string? feedsource)
-                                      (URL. feedsource)
-                                      feedsource))))
+   (parse-internal (XmlReader. (string->url feedsource))))
   ([feedsource content-type]
-     (parse-internal (new XmlReader feedsource content-type))))
+   (parse-internal (XmlReader. (string->url feedsource) content-type)))
+  ([deefsource content-type lenient]
```
Comment (sir-pinecone) [inline src/feedparser_clj/core.clj]: @Gonzih you have a typo here and below. Should be `feedsource` instead of `deefsource`.

Comment (Gonzih) [inline src/feedparser_clj/core.clj]: Thanks, fixed

**Comment by Gonzih** [comment]
Ugh, are you sure about merging this changeset? I started working on 
spec integration but its not completed yet. I forgot about this PR 
actually, just continued working on my own fork :)

**Comment by Gonzih** [comment]
Like I even renamed package at this point :/

**Pull request merged**